index.html

```diff
@@ -197,5 +197,22 @@ <h5>Skills</h5>
         </ul>
     </article>
  </section>
+
+ <footer class="contact">
+    <article>
+        <div class="C_Card">
+            <h3>Contact me</h3>
+            <p>If you have an application you are interested 
+                in developing, a feature that you need built or a 
+                project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/mnqwwwpj" method="POST">
+            <input type="text" name="name" placeholder="Enter your name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Enter your email" required>
+            <textarea name="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            <button type="submit" class="buttonBtn">Get It Touch</button>
```

Comment:
- [ ] Kindly make sure that the button has the same styling for (active, hover, pressed, ...), to match the Figma template.
![image](https://user-images.githubusercontent.com/62290545/171226346-d32596ba-a7a3-412e-8c2a-38fe0b1a150b.png)